### ISSUE TITLE
Upgrading to 2.0 breaks acme.sh issued certs

### ISSUE THREAD

**Issue by knox** [Froxlor/Froxlor#1073]
**Describe the bug**
When upgrading Froxlor to 2.0 acme.sh's internal configuration files for previously issued certificates become invalid and thus they fail to renew from cron.

The cause is the breaking changed webroot directory:

```
# grep Le_Webroot /root/.acme.sh/*/*.conf
```

One simple way to update acme.sh's internal configuration files is to re-issue the certificates with the new webroot parameter. Unfortunately Froxlor's cron script for Let's Encrypt does not neccessarily triggers such re-issue.

**System information**
* Froxlor version: 2.0.6
* Web server: apache2
* OS/Version: debian 11.6

**To Reproduce**
1. Have a working installation < 2.0 with Let's Encrypt enabled domains
2. Upgrade to 2.0
3. Manually run acme.sh for a domain that needs renewal

**Expected behavior**
Upgrading Froxlor to 2.0 must not break renewal of previously issued Let's Encrypt certificates.

**Logfiles**
Error mail from cron:
```
[Wed 11 Jan 2023 12:04:16 AM CET] domain.com:Verify error:2a01:aaa:bbb:2342::1: Invalid response from http://domain.com/.well-known/acme-challenge/aaaa9aaanbgAAAvCQ3hFTXylUSP6orxFBSr1mU8aaaa: 404
[Wed 11 Jan 2023 12:04:16 AM CET] Please add '--debug' or '--log' to check more details.
[Wed 11 Jan 2023 12:04:16 AM CET] See: https://github.com/acmesh-official/acme.sh/wiki/How-to-debug-acme.sh
[Wed 11 Jan 2023 12:04:17 AM CET] Error renew domain.com.
```

**Additional context**

I was able to resolve my broken installation by manually performing this:
1. Alter Froxlor's database to mark all certificates for a re-issue
```
UPDATE domain_ssl_settings SET expirationdate=NULL
```
2. Manually run Froxlor's cron script for Let's Encrypt to trigger re-issue for all certificates
```
# /var/www/html/froxlor/bin/froxlor-cli froxlor:cron letsencrypt --force
```

**Comment by d00p** [comment]
We'll look into it, thank you

**Comment by d00p** [comment]
Working on a cli command to fix this. Would you care joining Discord for testing and discussion? It would be much appreciated

**Comment by d00p** [comment]
Fixed in 87bd80eea16f7b66329eb85ad5882ed217427d29
